generator/app/index.js

```diff
@@ -0,0 +1,112 @@
+"use strict";
+
+module.exports = {
+  description: "Create app and route files",
+  prompts: [
+    {
+      type: "input",
+      name: "name",
+      message: "Name of the app",
+      validate: (value) => {
+        if (/.+/.test(value)) {
+          return true;
+        }
+        return "name is required";
+      },
+    },
+    {
+      type: "input",
+      name: "routeName",
+      message: "Name of the route",
+      validate: (value) => {
+        if (/.+/.test(value)) {
+          return true;
+        }
+        return "route is required";
+      },
+      description: "This is used in router.js",
+    },
+    {
+      type: "confirm",
+      name: "InjectRepository",
+      default: false,
+      message: "Would you inject repository in the methods?",
+    },
+    {
+      type: "input",
+      name: "repositoryName",
+      message: "Name of existing repository.",
+      description: "Only needed if previous answer was in affirmitive?",
```

Comment:
We can use this [plugin](https://github.com/plopjs/plop/blob/master/inquirer-prompts.md), so we don't have to show the next question if it depends on the previous one.